### ISSUE TITLE
Regarding training

### ISSUE THREAD

**Issue by skaasr** [JK-the-Ko/PEIPNet#2]
Hello, if I interrupt the training process halfway and have saved some of the weights, what command should I use to resume the training?

**Comment by JK-the-Ko** [comment]
There is no code for resuming the training. 
You should load the weights, and if cosine annealing learning rate decay is applied, you need to find the learning rate at the corresponding iteration and resume training.

**Comment by skaasr** [comment]
Of course, thank you very much for taking your valuable time to respond to me! I appreciate it!
